apis/quarkus-common/src/main/java/io/stargate/sgv2/common/grpc/StargateBridgeClient.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.common.grpc;
+
+import io.stargate.bridge.proto.QueryOuterClass.Batch;
+import io.stargate.bridge.proto.QueryOuterClass.Query;
+import io.stargate.bridge.proto.QueryOuterClass.Response;
+import io.stargate.bridge.proto.Schema;
+import io.stargate.bridge.proto.Schema.CqlKeyspaceDescribe;
+import io.stargate.bridge.proto.Schema.CqlTable;
+import io.stargate.bridge.proto.Schema.SchemaRead;
+import io.stargate.bridge.proto.Schema.SupportedFeaturesResponse;
+import io.stargate.sgv2.common.futures.Futures;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+
+/**
+ * The client that allows Stargate services to communicate with the "bridge" to the persistence
+ * backend.
+ */
+public interface StargateBridgeClient {
```

Comment:
The reason this is here is because I plan to reimplement it in terms of the Mutiny bridge. This will provide a temporary solution for a quick migration of the existing code, and for GraphQL we're going to need some kind of wrapper anyway (graphql-java doesn't support reactive).